docs/build/program/apis/data-client.md

```diff
@@ -846,3 +846,232 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
+
+**Returns:**
+
+- None.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient.delete_dataset).
+
+```python {class="line-numbers linkable-line-numbers"}
+await data_client.delete_dataset(
+  id="<dataset-id>"
+)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### AddBinaryDataToDatasetByIds
+
+Add the [BinaryData](https://python.viam.dev/autoapi/viam/proto/app/data/index.html#viam.proto.app.data.BinaryData) to the provided dataset.
+This BinaryData will be tagged with the VIAM_DATASET\_{id} label.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `binary_ids` (List[[BinaryID](https://python.viam.dev/autoapi/viam/gen/app/data/v1/data_pb2/index.html#viam.gen.app.data.v1.data_pb2.BinaryID "viam.gen.app.data.v1.data_pb2.BinaryID")]): The IDs of binary data to add to dataset.
+- `dataset_id` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The ID of the dataset to be added to.
+
+**Returns:**
+
+- None.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient.add_binary_data_to_dataset_by_ids).
+
+```python {class="line-numbers linkable-line-numbers"}
+from viam.proto.app.data import BinaryID
+
+binary_metadata = await data_client.binary_data_by_filter(
+    include_file_data=False
+)
+
+my_binary_ids = []
+
+for obj in binary_metadata:
+    my_binary_ids.append(
+        BinaryID(
+            file_id=obj.metadata.id,
+            organization_id=obj.metadata.capture_metadata.organization_id,
+            location_id=obj.metadata.capture_metadata.location_id
+            )
+        )
+
+await data_client.add_binary_data_to_dataset_by_ids(
+    binary_ids=my_binary_ids,
+    dataset_id="<dataset-id>"
+)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### RemoveBinaryDataFromDatasetByIds
+
+Remove the BinaryData from the provided dataset.
+This BinaryData will lose the VIAM_DATASET\_{id} tag.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `binary_ids` (List[[BinaryID](https://python.viam.dev/autoapi/viam/gen/app/data/v1/data_pb2/index.html#viam.gen.app.data.v1.data_pb2.BinaryID "viam.gen.app.data.v1.data_pb2.BinaryID")]): The IDs of binary data to remove from dataset.
+- `dataset_id` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The ID of the dataset to be removed from.
+
+**Returns:**
+
+- None.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient.remove_binary_data_from_dataset_by_ids).
+
+```python {class="line-numbers linkable-line-numbers"}
+from viam.proto.app.data import BinaryID
+
+binary_metadata = await data_client.binary_data_by_filter(
+    include_file_data=False
+)
+
+my_binary_ids = []
```

Comment:
Is this meant to be left empty?

Reply:
read the following lines (yes, IDs are being appended to the array there)